build_minecraft_bot.sh

```diff
@@ -20,12 +20,21 @@ if [ -d catkin_ws/src/minecraft_bot ] ; then
 else
     cd catkin_ws/src
     # Add the package
-    ln -s ../../minecraft_bot/ minecraft_bot
+    catkin_create_pkg minecraft_bot std_msgs rospy roscpp
+    cp -R ../../minecraft_bot .
+    # ln -s ../../minecraft_bot/ minecraft_bot    Symlinks do not work in virtualbox natively
```

Comment:
When using vagrant, the symlink command gives this error:

ln: failed to create symbolic link ‘minecraft_bot’: Protocol error

This happens when using a shared folder in virtualbox.

Found a fix I'm testing:
http://www.ahtik.com/blog/fixing-your-virtualbox-shared-folder-symlink-error/


Give me 5 mins..



On Sun, Jan 10, 2016 at 11:21 PM, Amen Belayneh <notifications@github.com>
wrote:

> In build_minecraft_bot.sh
> <https://github.com/opencog/opencog-to-minecraft/pull/17#discussion_r49289319>
> :
>
> > @@ -20,12 +20,21 @@ if [ -d catkin_ws/src/minecraft_bot ] ; then
> >  else
> >      cd catkin_ws/src
> >      # Add the package
> > -    ln -s ../../minecraft_bot/ minecraft_bot
> > +    catkin_create_pkg minecraft_bot std_msgs rospy roscpp
> > +    cp -R ../../minecraft_bot .
> > +    # ln -s ../../minecraft_bot/ minecraft_bot    Symlinks do not work in virtualbox natively
>
> what do yo mean by symlinks don't work in virtualbox?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog-to-minecraft/pull/17/files#r49289319>.
>